src/brackets.js

```diff
@@ -162,50 +153,50 @@ define(function (require, exports, module) {
         // in the modules since they would run in context of the unit test window,
         // and would not have access to the app html/css.
         brackets.test = {
-            CodeHintManager         : CodeHintManager,
-            CodeInspection          : CodeInspection,
-            CommandManager          : CommandManager,
-            Commands                : Commands,
+            CodeHintManager         : require("editor/CodeHintManager"),
```

Comment:
Oh, I see. This PR is 3 months old, so I don't exactly know the initial motivation, but I _think_ it might be that this just looks cleaner to me, than having a mixture of `require`s and variable references.